styles/accessibility.css

```diff
@@ -0,0 +1,5 @@
+:focus-visible {
```

Comment:
Should this apply to everything or just `.nav-link:focus-visible`?

Reply:
everything. The reason we need !important is because of .nav-link:focus-visible

Reply:
This can cause a gap on adjacent elements like this: 
<img width="100" alt="Screenshot 2024-08-26 at 3 45 33 PM" src="https://github.com/user-attachments/assets/c3332663-2259-4a36-bf13-a00fd86040c9">

Reply:
Do you have a suggestion on what to apply it to because the original was buttons (which I assumed to include links with .btn class) and nav links

Reply:
@alundgard wants it on everything, but change the black to the shadow and white for the outline.

Reply:
Thanks! SODA approved this design as a general-purpose focus ring. They said the thickness of the rings doesn't matter: 1px black and 1px white is enough. But now that I see the screenshots above, we might want to bump the thickness to 2px each for greater emphasis.

Reply:
done.